config.toml

```diff
@@ -88,31 +88,15 @@ enableEmoji = true
   [[params.social]]
       icon      = "github" # icon name without the 'fa-'
       icon_pack = "fab"
-      url       = "https://github.com/apreshill/apero"
+      url       = "https://github.com/NewGraphEnvironment"
   [[params.social]]
-      icon      = "facebook-square" # icon name without the 'fa-'
-      icon_pack = "fab"
+      icon      = "envelope" # icon name without the 'fa-'
```

Comment:
keen to skip facebook.  made an account years ago to look for accommodations but never use it

Reply:
I actually meant to link the envelope with an email link like on the old website, I'll link it to the contact page